### DISCUSSION TITLE
Using useRouteLoaderData with createRemixStub

### DISCUSSION THREAD

**Discussion by imballinst** [remix-run/remix#8658] [Q&A]
Hello everyone, I am using https://github.com/kiliman/remix-flat-routes, and in my part of code I have this file called `routes/_admin/_layout.tsx` -- which, in the actual application, I can do this to retrieve the loader value from it:

```ts
useRouteLoaderData('routes/_admin/_layout')
```

I wonder if we could use createRemixStub with this `useRouteLoaderData`? Because doing this:

```tsx
const RemixStub = createRemixStub([
  {
    path: '/',
    id: 'root',
    index: true,
    Component() {
      return <div>Hello, world!</div>
    }
  },
  {
    path: '/teams',
    id: 'teams',
    Component() {
      return <Outlet />
    },
    children: [
      {
        path: '/teams/:teamId/players',
        id: 'players',
        Component() {
          const val = useRouteLoaderData('teams')

          return (
            <div>
              <span>val: {val}</span>xdd
            </div>
          )
        }
      }
    ]
  }
])

const { debug } = render(<RemixStub initialEntries={['/teams/1/players']} />)
// ^
// <body>
//   <div />
// </body>
```

Will result in the debug only returns this empty thing @_@ 

EDIT: before I embarrass myself further, I'll see if I can update Remix in my project to v2

**Comment by imballinst** [comment]
I think it's because my Remix is still v1 😅 I'll close this discussion. Will reopen if I face further issues about similar topic.

**Comment by mnemitz** [comment]
I'm using Remix V2 and wondering about the same thing
